XCUITests/FxScreenGraph.swift

```diff
@@ -332,4 +341,10 @@ extension Navigator {
             self.goto(TabTray)
         }
     }
+
+    func changeBrowserView(view: String) {
```

Comment:
If you pass the enum around, instead of the string: 

```swift
    func changeBrowserView(view: BrowserMenuChoice) {
    }
```

you can start doing: 

```swift
navigator.changeBrowserView(view: .desktop)
```

which means auto-complete works, and the compiler checks you haven't made any spelling mistakes, and if the production code changes (unlikely), then you change it in one place not loads.